src/SFML/Audio/SoundSource.cpp

```diff
@@ -204,5 +205,6 @@ SoundSource::Status SoundSource::getStatus() const
 
     return Stopped;
 }
+// NOLINTEND(readability-make-member-function-const)
```

Comment:
Is this needed in both the header and the implementation? Can we just put it in one or the other?

Reply:
It seems to need it in both places.